core/src/com/mygdx/game/client/input/GameScreenInputAdapter.java

```diff
@@ -48,8 +57,9 @@ private void loadGameObjects(Map<Coordinates, Entity> coordinatesEntityMap) {
   @Override
   public boolean touchDown (int screenX, int screenY, int pointer, int button) {
     GameObject selecting = getObject(screenX, screenY);
-    if (selecting != null)
-      System.out.println(selecting.coords);
+    if (selecting != null) {
+      ChooseUnitFieldDialog.createCustomDialog(parentStage, gameState.getFieldList().get(selecting.coords));
```

Comment:
Może zamienić teraz w GameState Map<Coordinates, Entity> na jakieś Map<Coordinates, List<Entity>> i ominąć na razie ten cały Slot/FieldComponent? Albo na odwrót i zastanowić się czy GameState ma faktycznie sens.

To może być do obgadania jakoś.

Reply:
Zostańmy na razie przy tym co jest, ogólnie myślę czy nie zrobić potem żeby mogło być więcej jednostek na polu. Potem to przemyślimy